tests/framework/extensions/prime/primechecks.go

```diff
@@ -0,0 +1,76 @@
+package primechecks
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	client "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/rancherversion"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	rancherBrand         = "suse"
+	PodResourceSteveType = "pod"
+)
+
+// CheckUIBrand checks the UI brand of Rancher Prime. If the Rancher instance is not Rancher Prime, the UI brand should be blank.
+func CheckUIBrand(client *rancher.Client, isPrime bool, brand *client.Setting) {
+	if isPrime {
+		if rancherBrand == brand.Value {
+			logrus.Infof("Rancher Prime UI brand: %s", rancherBrand)
+		}
+	} else {
+		logrus.Infof("Rancher UI brand is blank")
+	}
+}
+
+// CheckVersion checks the if Rancher Prime is set to true and the version of Rancher.
+func CheckVersion(isPrime bool, rancherVersion string, serverConfig *rancherversion.Config) {
+	if isPrime {
+		if isPrime == serverConfig.IsPrime {
+			logrus.Infof("Rancher Prime: %t | Version: %s", isPrime, rancherVersion)
+		}
+	} else {
+		logrus.Infof("Rancher Prime: %t | Version: %s", isPrime, rancherVersion)
+	}
+}
+
+// CheckSystemDefaultRegistry checks if the system default registry is set to the expected value.
+func CheckSystemDefaultRegistry(isPrime bool, primeRegistry string, registry *client.Setting) {
```

Comment:
Yeah, to be honest, I thought of just removing the function altogether. I kept it in since I figured it's a simple UI check, but I'm not opposed to just removing it.